### ISSUE TITLE
Resize window on high dpi monitors

### ISSUE THREAD

**Issue by mattrussellcrick** [saalfeldlab/bigwarp#118]
Hi,

Resizing windows on a high dpi monitor (built in MacBook Pro 13") can sometimes lead to the view getting distorted and stuck. I think this happens once you start doing stuff, after the window has been resized. I've attached the Console errors associated with this (I think!)

Cheers,

Matt

[BigWarp resize error.txt](https://github.com/saalfeldlab/bigwarp/files/5913374/BigWarp.resize.error.txt)

**Comment by bogovicj** [comment]
Do you experience this with bigdataviewer / imagej in general? Or bigwarp specifically?

**Comment by mattrussellcrick** [comment]
Actually, having just updated Fiji, I can't open the files I could open before, either with BigWarp, or with Open Current Image.

I have a really old version of java that I didn't get around to updating (that Fiji keeps complaining about when I run updates). Perhaps that's the issue. I've attached some error logs.
[BDV_currentimage1.txt](https://github.com/saalfeldlab/bigwarp/files/6030133/BDV_currentimage1.txt)
[BDV_BigWarp_Exception.txt](https://github.com/saalfeldlab/bigwarp/files/6030134/BDV_BigWarp_Exception.txt)

**Comment by bogovicj** [comment]
I have not run into this, so I'm optimistic that an update will have / did fix this.
So I'll close this, but do re-open this (or make a new issue) if it comes up again.
